src/sass/_advantages.scss

```diff
@@ -0,0 +1,86 @@
+// section styling - have to be adjusted on tablets and desktop
+.advantages {
+  background-color: $bottom-section-background;
+  margin: 0;
+  padding: 40px 20px;
+  min-width: 320px;
+
```

Comment:
Brakuje mi kontenera ustawionego na 320px.
Chodzi mi o to, że gdy jesteś w przeglądarce na inspektorze to przy rozszerzaniu mobilki cała Twoja sekcja jest "przyklejona do lewej strony. 
Przy rozszerzaniu ma być na środku :)